src/stdatamodels/jwst/transforms/models.py

```diff
@@ -1989,6 +1991,164 @@ def _evaluate_transform_guess_form(model, x=0, y=0, t=0):
     raise TypeError(f"Expected a model or list of models, but got {type(model)}. ")
 
 
+class MIRIWFSSBackwardDispersion(_BackwardGrismDispersionBase):
+    """Calculate the dispersion extent of MIRI WFSS pixels."""
+
+    def __init__(self, orders, lmodels=None, xmodels=None, ymodels=None, name=None):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        orders : list
+            The list of orders which are available to the model. For MIRIWFSS orders = 1.
+        lmodels : list[1D models]
+            The list inverse dispersion 1D polynomial models. Use to determine the trace parameter.
+        xmodels : list[1D models]
+            The list of 1D polynomial models in x. It depends on the trace parameter
+        ymodels : list[2D models]
+            The list of 2D polynomial models in y.
+            There are three 2D polynomials to correct for spatial dependence. These polynomials
+            depend on location on the detector and the trace parameter.
+        name : str, optional
+            Name of the model
+        """
+        if name is None:
+            name = "miri_wfss_backward_dispersion"
+        super().__init__(
+            orders,
+            lmodels=lmodels,
+            xmodels=xmodels,
+            ymodels=ymodels,
+            name=name,
+        )
+
+    def evaluate(self, x, y, wavelength, order):
+        """
+        Transform from the direct image plane to the dispersed plane.
+
+        Parameters
+        ----------
+        x, y : float
+            Input x, y location in the direct image
+        wavelength : float
+            Wavelength in microns
+        order : int
+            Input spectral order
+
+        Returns
+        -------
+        x, y : float
+            The x, y values in the dispersed plane.
+        x0 y0 : float
+            Source object x-center, y-center in the direct image. Same as input x,y
+        order : int
+            Output spectral order, same as input
+        """
+        wavelength = np.atleast_1d(wavelength)
+        x = np.atleast_1d(x)
+        y = np.atleast_1d(y)
+
+        if (wavelength < 0).any():
+            raise ValueError("Wavelength should be greater than zero")
+        try:
+            iorder = self._order_mapping[int(order.flatten()[0])]
+        except KeyError as err:
+            raise ValueError("Specified order is not available") from err
+        # t is trace normalization parameters (it has values of 0 to 1)
+
+        t = self.lmodels[iorder](wavelength)
```

Comment:
```suggestion

        # t is trace normalization parameters (it has values of 0 to 1)
        t = self.lmodels[iorder](wavelength)
```
spacing change to clarify which line the comment refers to